### PR TITLE
Update kubeflow/kubeflow readme

### PR DESCRIPTION
**Pull request by RFMVasconcelos**
Just starting this update as a placeholder so we can centralize contributions

### CODE REVIEW

File: README.md
Comment (andreyvelich) [inline README.md:16]: Should we add appropriate links to the WG in the community repo ?
E.g. for [Training](https://github.com/kubeflow/community/tree/master/wg-training).

File: README.md
Comment (Bobgy) [inline README.md:10]: This is great!
Can we also add main repos for each WG where they prefer for github issues?
and recommend people creating issues for individual repos

e.g. for Pipelines, I'd suggest putting down:
* kubeflow/pipelines
* kubeflow/kfp-tekton

/cc @animeshsingh

Comment (RFMVasconcelos) [reply README.md:10]: What would be a good structure? 

* [Pipelines](https://github.com/kubeflow/community/tree/master/wg-pipelines) - [Code](https://github.com/kubeflow/pipelines)

Or 

* Pipelines - [Working Group](https://github.com/kubeflow/community/tree/master/wg-pipelines) - [Code](https://github.com/kubeflow/pipelines)

?

File: README.md
Comment (thesuperzapper) [inline README.md:7]: I think move this description of working groups under the `## Working Groups` heading.

Comment (RFMVasconcelos) [reply README.md:7]: That's a good point! +1

Comment (RFMVasconcelos) [reply README.md:8]: Done

File: README.md
Comment (animeshsingh) [reply README.md:10]: @RFMVasconcelos 

this is the 2nd repo - https://github.com/kubeflow/kfp-tekton


Additionally @Bobgy couple of comments on this
1. Charter has disappeared from the WG folder https://github.com/kubeflow/community/tree/master/wg-pipelines
2. Would like to add my name there for KFP-Tekton project, so that folks know who to connect for that

File: README.md
Comment (PatrickXYS) [inline README.md:12]: We need to update those links to make sure they can persist for a long time without going back and forth to fix it.

File: README.md
Comment (thesuperzapper) [inline README.md:5]: I think we should give this a heading to draw the user's attention.

For example:
```md
---

## Documentation
Please refer to the official docs at [kubeflow.org](http://kubeflow.org).
```

**Review by thesuperzapper — CHANGES_REQUESTED** [review]

File: README.md
Comment (RFMVasconcelos) [reply README.md]: Done!

File: README.md
Comment (RFMVasconcelos) [reply README.md:14]: That is a good idea! What would be a good way to do it?

File: README.md
Comment (thesuperzapper) [inline README.md:13]: This link is missing a dash, it should be `https://github.com/kubeflow/community/tree/master/wg-deployment`

File: README.md
Comment (thesuperzapper) [reply README.md:12]: @RFMVasconcelos can't we just include that information in the `kubeflow/community` repo?

Keeping the current links you made, e.g:`https://github.com/kubeflow/community/tree/master/wg-notebooks`

**Review by thesuperzapper — CHANGES_REQUESTED** [review]

File: README.md
Comment (Bobgy) [reply README.md:14]: The current links look right. kubeflow/community/master is the long term location

**Review by Bobgy — COMMENTED** [review]
/lgtm

Looks great to me!
despite some minor mistakes people have pointed out.

File: README.md
Comment (thesuperzapper) [reply README.md:13]: @RFMVasconcelos  please fix this before merging 
/hold

**Review by terrytangyuan — APPROVED** [review]
Looks great! Thanks.

/lgtm

File: README.md
Comment (RFMVasconcelos) [reply README.md:12]: @thesuperzapper I agree, this way we only redirect here for simplicity.

File: README.md
Comment (RFMVasconcelos) [reply README.md]: Fixed!

**Comment by RFMVasconcelos** [comment]
@Bobgy @terrytangyuan Thank you for the review! Unfortunately, there was a broken link that I had missed. Just fixed it. 

Can you please review again?

**Review by terrytangyuan — APPROVED** [review]
/lgtm

File: README.md
Comment (PatrickXYS) [reply README.md:14]: > * [Deployment] (https://github.com/kubeflow/community/tree/master/wgdeployment)

This is not correct, can we make it with a dash? `wg-deployment`

**Comment by thesuperzapper** [comment]
@Bobgy we need you to `/approve` this

**Comment by Bobgy** [comment]
/approve
Thank you all for the efforts!

**Pull request merged**
